spec/models/issue_spec.rb

```diff
@@ -0,0 +1,118 @@
+require 'rails_helper'
+
+RSpec.describe Issue, type: :model do
+  before(:each) do
+    @issue = Issue.new(reporter_id: 321,
+                       name: 'Maria Ivanivna Polishchuk',
+                       address: '12 Chornovola street, 3, Kyiv, Ukraine',
+                       tel: '+38 (099) 22 22 4444',
+                       email: 'user@example.com',
+                       category: 'Healthcare',
+                       description: 'a' * 255,
+                       attachment: 'https://drive.google.com/path_to_the_attachment')
```

Comment:
https://github.com/carrierwaveuploader/carrierwave/wiki/How-to:-Use-test-factories

thsi is correct way to use attachments. No external urls or using external services

Also, use factory girl